File: CS/GridProject/ExtendedGridView/ExtendedGridColumnCollection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid.Columns;
using DevExpress.XtraGrid.Views.Base;

namespace ExtendedGridViewSpace
{
    public class ExtendedGridColumnCollection : GridColumnCollection
    {
        public ExtendedGridColumnCollection(ColumnView view)
            : base(view)
        { }

        protected override GridColumn CreateColumn()
        {
            return new ExtendedGridColumn();
        }
    }
}

File: VB/GridProject/ExtendedGridView/ExtendedGridColumnCollection.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid.Columns
Imports DevExpress.XtraGrid.Views.Base

Namespace ExtendedGridViewSpace
	Public Class ExtendedGridColumnCollection
		Inherits GridColumnCollection

		Public Sub New(ByVal view As ColumnView)
			MyBase.New(view)
		End Sub

		Protected Overrides Function CreateColumn() As GridColumn
			Return New ExtendedGridColumn()
		End Function
	End Class
End Namespace
